### ISSUE TITLE
FYI: Ansible 2.4 support

### ISSUE THREAD

**Issue by dmsimard** [willthames/ansible-lint#278]
# Issue Type
- Bug report

# Ansible and Ansible Lint details
```
ansible==2.4.0
ansible-lint==3.4.13
```
Installed from pip (tox)

# Actual Behaviour

This is just a FYI -- as 2.4 is slowly starting to settle in and start the freezing process, it might be time to start looking at making ansible-lint compatible with the upcoming release.

Currently seeing the following error [here](http://logs.openstack.org/70/494070/4/check/gate-ara-integration-py27-devel-centos-7-nv/d83b8c0/console.html#_2017-08-17_20_55_20_395044):
```
2017-08-17 20:55:20.395044 | + ansible-lint ara/tests/integration/include_role.yml
2017-08-17 20:55:21.066260 | Traceback (most recent call last):
2017-08-17 20:55:21.066542 |   File "/home/jenkins/workspace/gate-ara-integration-py27-devel-centos-7-nv/.tox/py27/bin/ansible-lint", line 7, in <module>
2017-08-17 20:55:21.066650 |     from ansiblelint.main import main
2017-08-17 20:55:21.066876 |   File "/home/jenkins/workspace/gate-ara-integration-py27-devel-centos-7-nv/.tox/py27/lib/python2.7/site-packages/ansiblelint/__init__.py", line 28, in <module>
2017-08-17 20:55:21.066957 |     import ansiblelint.utils
2017-08-17 20:55:21.067177 |   File "/home/jenkins/workspace/gate-ara-integration-py27-devel-centos-7-nv/.tox/py27/lib/python2.7/site-packages/ansiblelint/utils.py", line 53, in <module>
2017-08-17 20:55:21.067278 |     from ansible.plugins import module_loader
2017-08-17 20:55:21.067402 | ImportError: cannot import name module_loader
```
The playbook in question, while I'm sure is not very relevant to the issue, is [here](https://github.com/openstack/ara/blob/master/ara/tests/integration/include_role.yml).

**Comment by willthames** [comment]
Thanks @dmsimard.

This is now fixed in 3.4.14

**Comment by willthames** [comment]
Or 3.4.15 even (3.4.14 didn't actually contain the fix, oops)
